src/core/server/saved_objects/migrations/initial_state.ts

```diff
@@ -88,6 +88,7 @@ export const createInitialState = ({
     retryAttempts: migrationsConfig.retryAttempts,
     batchSize: migrationsConfig.batchSize,
     maxBatchSizeBytes: migrationsConfig.maxBatchSizeBytes.getValueInBytes(),
+    ignoreUnknownObjects: migrationsConfig.ignoreUnknownObjects === kibanaVersion,
```

Comment:
I wonder if we should output a message/warning if `migrations.ignoreUnknownObjects` is specified but does not match the current version?

Reply:
It makes sense!